packages/babel-preset/index.js

```diff
@@ -44,16 +44,13 @@ function addDefaultOptions(explicitOptions) {
     envCorejs: null,
     debug: false,
     targets: undefined, // Targets for @babel/preset-env.
+    useBuiltIns: false,
     ignoreBrowserslistConfig: false,
     configPath: '.',
     include: [],
     ...explicitOptions,
   };
 
-  if (options.envCorejs === null) {
```

Comment:
this next check takes care of this... all this does is give us warnings when using core js for runtime, but not using `useBuiltIns`